sktime/forecasting/base/_base.py

```diff
@@ -400,6 +435,7 @@ def fit(self, y, X=None, fh=None):
 
         # this should happen last
         self._is_fitted = True
```

Comment:
worth a thought, to avoid the duplication, is it maybe better to make `_is_fitted` a property too, and just have it default to the same thing as `is_fitted`? For downwards compatibility of the private variable.